### PR TITLE
Feature/581

### PR DESCRIPTION
**Pull request by Haytham1337**
Added Sizes Page on Admin.
image
image

### CODE REVIEW

File: src/pages/sizes/sizes.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  }, [sizesList]);
+
+  if (loading) {
+    return <LoadingBar />;
+  }
+  const sizeItems = map(sizesList, (size) => (
+    <TableContainerRow
+      showAvatar={false}
+      showEdit
+      name='Розмір для сумки'
```
Comment (Haytham1337) [inline src/pages/sizes/sizes.js:41]: this string is  temporarily

File: src/pages/sizes/sizes.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        >
+          {config.titles.sizesTitles.mainPageTitle}
+        </Typography>
+        <Button
+          id='add-sizes'
+          component={Link}
+          variant='contained'
+          color='primary'
+        >
+          {CREATE_SIZE_TITLE}
```
Comment (Kiyooooko) [inline src/pages/sizes/sizes.js:65]: if we can't add size, I recommend to remove it

File: src/pages/sizes/sizes.js (hunk 1 above)
Comment (Kiyooooko) [reply src/pages/sizes/sizes.js:41]: In future, you have to use value from database

File: src/pages/sizes/sizes.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        </Button>
+      </div>
+      {!loading ? (
+        <TableContainerGenerator
+          pagination
```
Comment (Kiyooooko) [inline src/pages/sizes/sizes.js:70]: you can add showEdit={false} to hide edit icon

File: src/pages/sizes/sizes.js (hunk 1 above)
Comment (Haytham1337) [reply src/pages/sizes/sizes.js:41]: this string is temporarily

File: src/pages/sizes/sizes.js (hunk 2 above)
Comment (Haytham1337) [reply src/pages/sizes/sizes.js]: removed add btn

File: src/pages/sizes/sizes.js (hunk 3 above)
Comment (Haytham1337) [reply src/pages/sizes/sizes.js:61]: Set showEdit={false}  and showDelete={false}

File: src/pages/sizes/sizes.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Typography, Button } from '@material-ui/core';
+
```
Comment (AndriyLymych) [inline src/pages/sizes/sizes.js:4]: remove empty line

File: src/redux/sizes/sizes.sagas.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import { takeEvery, call, put } from 'redux-saga/effects';
+
+import { getAllSizes } from './sizes.operations';
+
+import { setSizes, setSizesLoading, setSizesError } from './sizes.actions';
+
+import { GET_SIZES } from './sizes.types';
+
```
Comment (AndriyLymych) [inline src/redux/sizes/sizes.sagas.js:8]: remove unnessesary empty lines

File: src/pages/sizes/sizes.js (hunk 4 above)
Comment (Haytham1337) [reply src/pages/sizes/sizes.js:4]: fixed

File: src/redux/sizes/sizes.sagas.js (hunk 5 above)
Comment (Haytham1337) [reply src/redux/sizes/sizes.sagas.js:8]: fixed

**Review by AndriyLymych — APPROVED** [review]

**Review by AndriyLymych — DISMISSED** [review]

File: src/configs/routes.js [hunk 6]
```diff
@@ -25,6 +25,7 @@ const routes = {
   pathToAddContact: '/contacts/add',
   pathToOrders: '/orders',
   pathToComments: '/comments',
+  pathToSizes: '/sizes',
   pathToOrderItem: '/orders/:id',
```
Comment (kapoliub) [inline src/configs/routes.js:29]: add path for size editing by id like `/sizes/:id`
and for adding

Comment (Haytham1337) [reply src/configs/routes.js:31]: add paths

**Review by kapoliub — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Review by Kiyooooko — APPROVED** [review]

**Pull request merged**